### PR TITLE
functionality to add webhook settings through post back url

### PR DESCRIPTION
**Pull request by aashutoshbane**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default class RusticiSdk {
    * @param {String} client
    * @param {Object} config
    */
-  constructor(client: string = 'superagent', config: IClient) {
+  constructor(config: IClient, client: string = 'superagent') {
```
Comment (supachris28) [inline src/index.ts]: swapping these variables is a breaking change and unnecessary

File: src/interfaces/webhook-config-interface.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default interface IWebhookConfig {
```
Comment (supachris28) [inline src/interfaces/webhook-config-interface.ts]: this looks more default than just webhook-related, can be called something like IConfigItem

File: src/interfaces/rustici-webhook-config-list.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import IWebhookConfig from './webhook-config-interface';
```
Comment (supachris28) [inline src/interfaces/rustici-webhook-config-list.ts]: This can be more generic than jsut webhook based, can be something like IConfig

**Review by supachris28 — CHANGES_REQUESTED** [review]

**Review by supachris28 — APPROVED** [review]

**Review by supachris28 — CHANGES_REQUESTED** [review]
Needs a version bump

**Review by supachris28 — APPROVED** [review]

**Review by ukmadlz — APPROVED** [review]

**Pull request merged**